src/compiler/sys.ts

```diff
@@ -456,7 +456,7 @@ namespace ts {
         getMemoryUsage?(): number;
         exit(exitCode?: number): void;
         realpath?(path: string): string;
-        /*@internal*/ getEnvironmentVariable(name: string): string | undefined;

```

Comment:
Isn't this going to be confusing if external clients use `undefined` to represent a missing variable but we use `""`?